BASE/src/ServerTelemetryChannel/Implementation/TelemetrySerializer.cs

```diff
@@ -27,6 +29,33 @@ public Uri EndpointAddress
         }
 
         public virtual void Serialize(ICollection<ITelemetry> items)
+        {
+            this.HandleTelemetryException(items);
+
+            var transmission = new Transmission(this.EndpointAddress, items);
+            this.Transmitter.Enqueue(transmission);
+        }
+
+        public virtual async Task<bool> Serialize(ICollection<ITelemetry> items, bool manualFlushAsync)
```

Comment:
Task-returning methods should have "Async" suffix. i.e `SerializeAsync`.
Also, please add a doc-comment explaining the meaning of `manualFlushAsync`.